radix-engine-lib/src/math/integer/bits.rs

```diff
@@ -13,17 +13,6 @@ macro_rules! checked_int_impl_large {
                 impl $t {
                     /// Returns the number of ones in the binary representation of `self`.
                     ///
-                    /// # Examples
-                    ///
-                    /// Basic usage:
-                    ///
-                    /// ```
-                    #[doc = "use scrypto::prelude::*;"]
-                    ///
-                    #[doc = "let n = " $t "::by(0b01001100u8);"]
-                    ///
-                    /// assert_eq!(n.count_ones(), 3);
-                    /// ```
```

Comment:
Possibly to make them compile? The documentation will be re-exported to scrypto and might be useful for developers. Also, they're also used for testing if I'm not mistaken.